pkg/test/expectations/expectations.go

```diff
@@ -457,6 +457,12 @@ func FindMetricWithLabelValues(name string, labelValues map[string]string) (*pro
 	return nil, false
 }
 
+func ExpectMetricGaugeValue(metricName string, expectedValue float64, labels map[string]string) {
+	metric, ok := FindMetricWithLabelValues(metricName, labels)
```

Comment:
Should you add `GinkgoHelper()` to the top of this since this is a helper function and we want to make sure the call stack returns correctly.

Reply:
Good catch